src/Sulu/Bundle/PageBundle/Trash/PageTrashItemHandler.php

```diff
@@ -104,13 +104,16 @@ public function store(object $page, array $options = []): TrashItemInterface
                 : $localizedPage->getExtensionsData();
 
             $pageTitles[$locale] = $localizedPage->getTitle();
+            $lastModified = $localizedPage->getLastModified() ? $localizedPage->getLastModified()->format('c') : null;
 
             $data['locales'][] = [
                 'title' => $localizedPage->getTitle(),
                 'locale' => $locale,
                 'creator' => $localizedPage->getCreator(),
                 'created' => $localizedPage->getCreated()->format('c'),
                 'author' => $localizedPage->getAuthor(),
+                'lastModifiedEnabled' => $localizedPage->getLastModifiedEnabled(),
```

Comment:
This should not be required to be part of trash. As the value should always be calculated based on the `lastModified` and not be saved by itself somewhere:

```suggestion
```